libs/components/datetime/src/lib/modules/datepicker/datepicker-calendar.component.ts

```diff
@@ -110,8 +113,8 @@ export class SkyDatepickerCalendarComponent {
       this.selectedDate = value;
       this.datepicker?.select(value, false);
     } else {
-      this.selectedDate = new Date();
-      this.datepicker?.select(new Date(), false);
+      this.selectedDate = this.startAtDate ?? new Date();
```

Comment:
Do we want to validate the date is valid using `this.#formatter.dateIsValid`?

Reply:
I don't think so, since `startAtDate` doesn't require validation or affect other properties' validations.